react/features/welcome/components/WelcomePage.native.js

```diff
@@ -285,23 +343,16 @@ class WelcomePage extends AbstractWelcomePage {
         return (
             <LocalVideoTrackUnderlay style = { styles.welcomePage }>
                 <View style = { _headerStyles.page }>
-                    <Header style = { styles.header }>
-                        <TouchableOpacity onPress = { this._onShowSideBar } >
-                            <Icon
-                                src = { IconMenu }
-                                style = { _headerStyles.headerButtonIcon } />
-                        </TouchableOpacity>
-                        <VideoSwitch />
-                    </Header>
                     <SafeAreaView style = { styles.roomContainer } >
                         <View style = { styles.joinControls } >
                             <Text style = { styles.enterRoomText }>
                                 { t('welcomepage.roomname') }
                             </Text>
+                            {/* // $FlowExpectedError*/}
```

Comment:
???

Reply:
Cannot create TextInput element because property autoComplete is missing in object type [1] but exists in props [2].